tests/test_endtoend.py

```diff
@@ -369,3 +338,121 @@ def probe_also_proxy(probe_result, hostname):
     probe_result.write("probe-also-proxy {}".format(hostname))
     success, request_ip = loads(probe_result.read())
     return success, IPv4Address(request_ip)
+
+
+def _get_swap_result(probe):
+    if probe.operation.name != "swap":
+        pytest.skip("Test only applies to --swap-deployment usage.")
+
+    # Telepresence won't try to swap anything back until it believes its job
+    # is done.  So make sure its job is done before we make any assertions
+    # about whether things were swapped back.
+    return probe.ensure_dead()
```

Comment:
The name of that method is confusing, even here when being used. The name should express that this is like `probe.get_result()` with an additional guarantee. How about `probe.get_result_after_dead()`?